internal/config/default_config.go

```diff
@@ -8,3 +8,14 @@ package config
 // This value is controlled by the --unpropagated-annotation command line, which may be set multiple
 // times.
 var UnpropagatedAnnotations []string
+
+// ExclusionByLabelsSpec specifies a label Key and Value which will cause an object to be excluded
+// from propagation if the object defines that label with this specific value.
+type ExclusionByLabelsSpec struct {
```

Comment:
Suggestion: `NoPropagationLabel` and then the variable is simply `NopropagationLabels` or something like that.